device_policy.py

```diff
@@ -106,28 +113,28 @@ def items_from_prefix(self, target_dict, prefix):
       returned_dict[key] = value
     return returned_dict
   
-  #convert the list of local accounts so it can be imported by protobuf
+  # Convert the list of local accounts so it can be imported by protobuf.
   def convert_local_accounts(self, policy_value):
     new_accounts = []
     
     for account in policy_value:
-      #setup common attributes
+      # Setup common attributes.
       new_account = {
-        "account_id": account["id"] + "@kiosk-apps",
+        "account_id": account["account_id"] + "@kiosk-apps",
         "type": account["type"]
       }
       info_prefix = None
       info_key = None
 
-      #get prefix and proto key from type
-      if account["type"] == 1:
+      # Get prefix and proto key from type.
+      if account["type"] == "ACCOUNT_TYPE_KIOSK_APP":
         info_prefix = "kiosk_"
         info_key = "kiosk_app"
-      elif account["type"] == 4:
+      elif account["type"] == "ACCOUNT_TYPE_WEB_KIOSK_APP":
         info_prefix = "web_kiosk_"
         info_key = "web_kiosk_app"
       
```

Comment:
These changes are all pointless and break compatibility with the original policies.json format.

Reply:
They were to have compatibility with the protobufs